dataframe-excel/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/xlsx.kt

```diff
@@ -93,17 +97,21 @@ public fun DataFrame.Companion.readExcel(
     sheetName: String? = null,
     skipRows: Int = 0,
     columns: String? = null,
+    formattingOptions: FormattingOptions? = null,
```

Comment:
If this argument is just a "range of columns to read as string", "formattingOptions" is not a good name. it should be something like "stringColumns". That said, if formattingOptions can do something more, like define other colTypes as well, then the name is better, but I would try to make the API similar to csv, where you can also define colTypes.